cmd/stayrtr/stayrtr.go

```diff
@@ -208,7 +208,7 @@ func processData(vrplistjson []prefixfile.VRPJson,
 	// grab the current time every time it's invoked, time calls can be slow on
 	// some platforms, so lets just get the unix time when we start and use that
 	// to compare it all
-	NowUnix := time.Now().UTC().Unix()
```

Comment:
Why did you drop the UTC? If the time zone changes on the machine the number will not be comparable?

Reply:
It’s implied in `.Unix()`, no?